### PR TITLE
dynamically construct paths for settings instead of expecting and har…

### PR DESCRIPTION
**Pull request by thraxil**
…d-coding /var/www/myapp/...

### CODE REVIEW

File: ccnmtldjango/template/+package+/settings_shared.py_tmpl [hunk 1]
```diff
@@ -90,7 +100,6 @@ MIDDLEWARE_CLASSES = (
 ROOT_URLCONF = '${package}.urls'
 
 TEMPLATE_DIRS = (
-    "/var/www/${package}/templates/",
     os.path.join(os.path.dirname(__file__), "templates"),
```
Comment (sdreher) [inline ccnmtldjango/template/+package+/settings_shared.py_tmpl]: replace os.path.dirname(__file__) with CURRENT_DIR?

File: ccnmtldjango/template/+package+/settings_production.py_tmpl [hunk 2]
```diff
@@ -3,13 +3,13 @@ from settings_shared import *
 import os
 
 TEMPLATE_DIRS = (
-    os.path.join(os.path.dirname(__file__), "templates"),
```
Comment (sdreher) [inline ccnmtldjango/template/+package+/settings_production.py_tmpl]: General comment: Do these settings (TEMPLATE_DIRS, MEDIA_ROOT, STATICMEDIA_MOUNTS) need overridden in production & staging settings when they are now generically constructed in settings_shared?

Comment (thraxil) [inline ccnmtldjango/template/+package+/settings_production.py_tmpl]: Good point. Yeah, I think we can take advantage of this to shrink staging/prod configs a bit.

**Comment by thraxil** [comment]
Took those out of settings_staging and settings_production. Also just removed the `STATICMEDIA_MOUNTS`, which AFAICT, is not used anywhere anymore.

**Pull request merged**